### ISSUE TITLE
Browser input - Quo2 Component

### ISSUE THREAD

**Issue by cammellos** [status-im/status-react#15972]
Create the quo2 component browser input as per designs:

https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=4840-135087&t=rY9tKnH8a9ROUrMe-0

Should be implemented according to best practices
https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md

Include component spec tests
https://github.com/status-im/status-mobile/blob/develop/doc/component-tests-overview.md

Communicate with design team before starting to make sure there is no unknown requirements and to understand the interactions

Blur variant is out of scope

**Comment by BalogunofAfrica** [comment]
@cammellos Figma link is broken

**Comment by cammellos** [comment]
@BalogunofAfrica we need to give you access, could you reach out to us on discord with your email adress?

Thank you!

**Comment by BalogunofAfrica** [comment]
Alright, can I get your discord username or can I ask Siddhart to help with this? @cammellos

**Comment by cammellos** [comment]
@BalogunofAfrica you can ping @siddarthkay , he's going to be able to help you
